pkg/resource/backup/sdk.go

```diff
@@ -144,8 +144,6 @@ func (rm *resourceManager) newDescribeRequestPayload(
 
 	if r.ko.Status.ACKResourceMetadata != nil && r.ko.Status.ACKResourceMetadata.ARN != nil {
 		res.SetBackupArn(string(*r.ko.Status.ACKResourceMetadata.ARN))
-	} else {
```

Comment:
It was removed by the code-generator since it's block of code that is not executed anymore https://github.com/aws-controllers-k8s/code-generator/commit/25c43e827527b43c652b6e1995265c8d6027567f